### PR TITLE
Improved and fixed tests in wrapper

### PR DESCRIPTION
**Pull request by ilyapuchka**

### CODE REVIEW

**Comment by kompozer** [comment]
@ilyapuchka Why do we need a `name` parameter for `CBCreateTemporaryDirectoryAndReturnURL`? How is it used?

File: Cryptobox/CryptoboxTests/CBBasicCryptoTest.m [hunk 1]
```diff
@@ -21,46 +21,47 @@ @implementation CBBasicCryptoTest
...
-    NSData *cipherData = [aliceSession encrypt:plainData error:&error];
+    NSData *cipherData = [aliceToBobSession encrypt:plainData error:&error];
     XCTAssertNil(error);
     XCTAssertNotNil(cipherData);
     XCTAssertNotEqual(plainData, cipherData);
     
-    CBSession *bobSession = nil;
-    CBSessionMessage *bobSessionMessage = [self.bobBox sessionMessageWithId:@"bob" fromMessage:cipherData error:&error];
+    //Bob's side
+    CBSessionMessage *bobToAliceSessionMessage = [self.bobBox sessionMessageWithId:@"sessionToAllice" fromMessage:cipherData error:&error];
```
Comment (kompozer) [inline Cryptobox/CryptoboxTests/CBBasicCryptoTest.m]: minor typo

File: Cryptobox/CryptoboxTests/CBTestCase.m [hunk 2]
```diff
@@ -7,12 +7,12 @@
 
 #import "Cryptobox.h"
 
-NSURL *__nullable CBCreateTemporaryDirectoryAndReturnURL(void);
+NSURL *__nullable CBCreateTemporaryDirectoryAndReturnURL(NSString *name);
 
-NSURL *__nullable CBCreateTemporaryDirectoryAndReturnURL(void)
+NSURL *__nullable CBCreateTemporaryDirectoryAndReturnURL(NSString *name)
```
Comment (kompozer) [inline Cryptobox/CryptoboxTests/CBTestCase.m]: Why not just add `static` here? Saves the extra declaration line

Comment (ilyapuchka) [inline Cryptobox/CryptoboxTests/CBTestCase.m]: Actually we can just remove it cause it's already defined as extern in header file

File: Cryptobox/CryptoboxTests/CBBasicCryptoTest.m [hunk 3]
```diff
@@ -70,21 +71,21 @@ - (void)testThatBasicTestCanRun
...
+    [self.bobBox closeSession:bobToAliceSession];
     
-    aliceSession = [self.aliceBox sessionById:@"alice" error:&error];
+    aliceToBobSession = [self.aliceBox sessionById:@"sessionWithBob" error:&error];
     XCTAssertNil(error);
-    XCTAssertNotNil(aliceSession);
+    XCTAssertNotNil(aliceToBobSession);
     
-    bobSession = [self.bobBox sessionById:@"bob" error:&error];
+    bobToAliceSession = [self.bobBox sessionById:@"sessionToAllice" error:&error];
```
Comment (kompozer) [inline Cryptobox/CryptoboxTests/CBBasicCryptoTest.m]: Another typo here, which causes the tests to fail

**Pull request merged**